server/monitor/src/main/java/org/apache/accumulo/monitor/rest/tables/TablesResource.java

```diff
@@ -112,6 +112,7 @@ private static TablesList generateTables(TablesList tableNamespace) {
     for (Entry<String,Table.ID> entry : Tables.getNameToIdMap(HdfsZooInstance.getInstance()).entrySet()) {
       String tableName = entry.getKey();
       Table.ID tableId = entry.getValue();
+      String canonicalTableId = tableId.canonicalID();
```

Comment:
I was thinking this could be deferred to the constructors of `TableInformation`; that way, this class wouldn't need to change at all, and we can keep the `Table.ID` object around as long as possible before it is serialized in `TableInformation`.

Reply:
@ctubbsii I implemented this, keeping `TablesResource` the way it was and having `TableInformation` still take the `Table.ID` object as a parameter, but the constructor takes the `canonicalID` from it to store.